src/utils/APIservices/PlioAPIService.js

```diff
@@ -11,10 +11,10 @@ export default {
           userId
       );
       resolve(response);
-      })
+    });
```

Comment:
Yo make your pre-commit actually pre-commit like Aman said!

Reply:
Yes did that
Due to some issue I had to delete and clone the repo again and then forgot to install pre-commit
But it's set up now, so this issue won't come up again